index.rst

```diff
@@ -178,8 +178,9 @@ Data of interest in the LFA originates from:
 
 Like the data from the Auxillary Telescope, users want to interact with the LFA data through the butler, an appropriate dataset type having been define. Moreover users require these data with very low latencies as it is likely that they need it in order to make on-the-fly adjustments to systems during commissioning.
 
-We therefore propose that the LFA Announcement Stream is monitored by the DM-CS and when data from these enumerated sources is made available, that it be injected into the data backbone, from whence it will be treated like data (and not telemetry) by upstream services.
+We therefore propose that the LFA Announcement Stream is monitored by a service running at the base.  The Archiver is a likely candidate, but could also be a new service specifically for this purpose (See  `LDM-152 <https://ls.st/LDM-152>`_ :cite:`LDM-152` to see how the Archiver fits into the overall middleware architecture.) and when data from these enumerated sources is made available, that it be injected into the data backbone, from whence it will be treated like data (and not telemetry) by upstream services.
```

Comment:
Change the link so it's _anonymous_. I this is necessary because both the `cite` command and the link are creating targets called `LDM-152`:

```
(See  `LDM-152 <https://ls.st/LDM-152>`__ :cite:`LDM-152` to see how the Archiver fits into the overall middleware architecture.)
```

Note double underscore following the link.